phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
why is the low freq specified in samples and the high freq in seconds?... that seems odd